regex-assembly/933210.ra

```diff
@@ -0,0 +1,31 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
+##! Helpers
+##!> define single_or_double_quotes ['"]
+##!> define string_in_round_brackets \(.+\)
+
+##!$ ;
+
+##!> assemble
+    ##! example payload: (system)(ls);
+    {{string_in_round_brackets}}{{string_in_round_brackets}}
+
+    ##! example payload: (sys)"tem"(ls);
+    {{string_in_round_brackets}}{{single_or_double_quotes}}[a-zA-Z-_0-9]+{{single_or_double_quotes}}{{string_in_round_brackets}}
```

Comment:
Can you use case-insensitive matches? Then replace `[a-zA-Z-_0-9]` with `[a-z0-9_-]`.
BTW, the dash should be in the last position anyway (according to our docs), IIRC.

Reply:
done